### PR TITLE
allow xunit reporter options to specify a title for the test groups

### PR DESCRIPTION
**Pull request by elangelo**

### CODE REVIEW

**Comment by boneskull** [comment]
thanks

File: lib/reporters/xunit.js [hunk 1]
```diff
@@ -65,7 +69,7 @@ function XUnit (runner, options) {
 
   runner.on('end', function () {
     self.write(tag('testsuite', {
-      name: 'Mocha Tests',
+      name: self.title ? self.title : 'Mocha Tests',
```
Comment (ScottFreeCode) [inline lib/reporters/xunit.js]: If I'm not mistaken, this is equivalent to `self.title || 'Mocha Tests'`.

File: lib/reporters/xunit.js [hunk 2]
```diff
@@ -51,6 +51,10 @@ function XUnit (runner, options) {
     self.fileStream = fs.createWriteStream(options.reporterOptions.output);
   }
 
+  if (options && options.reporterOptions && options.reporterOptions.title) {
+    self.title = options.reporterOptions.output;
```
Comment (dasilvacontin) [inline lib/reporters/xunit.js]: I think you meant `options.reporterOptions.title` here?

Comment (dasilvacontin) [reply lib/reporters/xunit.js]: Otherwise I'm missing context.

**Comment by boneskull** [comment]
old

**Pull request closed without merging**